### PR TITLE
Hippo export tweaks

### PR DESCRIPTION
**Pull request by asmega**
- fixes hippo pdf export
- add timeout and exception handling to rackspace upload

### CODE REVIEW

File: lib/hippo_file.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@ def filename
   end
 
   def blob
-    binary_blob = (find_property('hippo:text') || find_property('jcr:data')).value
+    binary_blob = (find_property('jcr:data') || find_property('hippo:text')).value
```
Comment (mconnell) [inline lib/hippo_file.rb]: Wondering if it's worth having a test that shows the precedence for the hippo blob?

**Comment by mconnell** [comment]
Other than the inline comment :+1:

**Comment by darokel** [comment]
:+1:

File: lib/hippo_file.rb (hunk 1 above)
Comment (asmega) [inline lib/hippo_file.rb]: There wasn't an original spec for this and I've not added one. I've only tested by actually running the script.

**Pull request merged**